synchronizer/l1_worker_etherman.go

```diff
@@ -161,9 +161,15 @@ func (w *workerEtherman) asyncRequestRollupInfoByBlockRange(ctx contextWithCance
 			case <-time.After(sleepBefore):
 			}
 		}
+
+		// Uncomment these lines to respond with a nil result to generate fast responses (just for develop!)
+		//w.setStatus(ethermanIdle)
+		//ch <- newResponseRollupInfo(nil, time.Second, typeRequestRollupInfo, &rollupInfoByBlockRangeResult{blockRange, nil, nil, nil})
+
```

Comment:
Do we need this code?

Reply:
is a fast reminder to do some tests to the synchronization methods